package/installation.xml

```diff
@@ -40,8 +40,11 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
         <!-- the default preselected modules in offline installation -->
         <default_modules config:type="list">
             <default_module>sle-module-basesystem</default_module>
+            <default_module>sle-module-desktop-applications</default_module>
+            <default_module>sle-module-development-tools</default_module>
             <default_module>sle-module-hpc</default_module>
-            <default_module>sle-module-python2</default_module>
```

Comment:
JFYI: The Python2 module was dropped in SP5, there is the Python3 module and it is not selected by default by SCC.